readers/mongodb/messages.go

```diff
@@ -109,6 +110,34 @@ func fmtCondition(chanID string, query map[string]string) *bson.D {
 			"name",
 			"protocol":
 			filter = append(filter, bson.E{Key: name, Value: value})
+		case "v":
```

Comment:
Can you try using values without parsing (because the value type in `bson.E` is `interface{}`)? Or at least consider returning conversion error.

Reply:
I agree, but as explained here #1061 the idea would be to replace `query map[string]string` by `rpm readers.PageMetadata`. That way the parsing would be done at the endpoint layer instead of here. If I return an error here I'll remove in my next PR.

Reply:
Does this mean that you will detect the invalid query at the endpoint layer? If yes, than this error checking should be removed.

Reply:
Yes, it will be previously detected to properly set the structure `PageMetadata`. For now I check it and `continue` to avoid a bug. I'll remove it for all readers in the next PR.